apps/condo/domains/notification/schema/RegisterPushTokenService.test.js

```diff
@@ -0,0 +1,60 @@
+// noinspection DuplicatedCode
+
+/**
+ * Generated by `createservice notification.RegisterPushTokenService --type mutations`
+ */
+
+const faker = require('faker')
+
+const { makeLoggedInAdminClient, makeClient, makeLoggedInClient } = require('@core/keystone/test.utils')
+
+const { expectToThrowAccessDeniedErrorToObj, expectToThrowAuthenticationErrorToObjects } = require('@condo/domains/common/utils/testSchema')
+
+const { registerPushTokenByTestClient } = require('@condo/domains/notification/utils/testSchema')
+const { DEVICE_SERVICE_TYPES_KEYS } = require('@condo/domains/notification/constants')
+const { getRandomItem } = require('@core/keystone/test.utils')
+
+const getRandomServiceType = () => getRandomItem(DEVICE_SERVICE_TYPES_KEYS)
+const getRandomTokenData = () => ({
+    deviceId: faker.datatype.uuid(),
+    token: faker.datatype.uuid(),
+    serviceType: getRandomServiceType(),
+})
+
+describe('RegisterPushTokenService', () => {
```

Comment:
Please, test a cases:
- when push token is already exist, it should be updated
- otherwise, it should be created.

Reply:
Yeah, more tests are coming.